docs/concepts/vega-chain/transactions.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 4
+title: Transactions
+vega_network: TESTNET
+hide_title: false
+---
+
+import NetworkParameter from '@site/src/components/NetworkParameter';
+
+A transaction is an instruction (command) issued the network, bundled with a signature, the submitter’s public key, a small proof-of-work calculation, and the target block height. A transaction that doesn't pass [validation](#validation) (for example, an order without enough assets to cover it) will be rejected.
+
+:::tip Viewing transactions
+Use the [Vega block explorer](https://explorer.fairground.wtf) to see transactions and their status, plus lots more data about the network.
+:::
+
+When a transaction is submitted, it goes to the validators’ mempools - which is is an organised queue where the transactions are stored, sorted, and verified before being added to a newly started block. A validated transaction enters a block based on the priority assigned to the transaction type, and the order in which it arrived. Vega does not have any gas-type charges that can boost priority or cause a transaction to be lower priority.
+
+## Commands
+A command is the specific instruction issued to the network within a transaction.
+
+The commands that non-validator users can be submitted to the network include submitting, amending, and cancelling orders, liquidity provisions and transfers, associating tokens and nominating validators, governance proposals and voting on proposals, and depositing and withdrawing assets.
```

Comment:
```suggestion
The commands that non-validator users can submit to the network include submitting, amending, and cancelling orders, liquidity provisions and transfers, associating tokens and nominating validators, governance proposals and voting on proposals, and depositing and withdrawing assets.
```

Reply:
Minor (but is likely just me so feel free to ignore): maybe "placing, amending, and cancelling orders" just because to be the the double use of "submit" in "can submit to the network include submitting" jars a bit for me?

Reply:
Yes - agreed, it's pretty ugly, will fix.